### DISCUSSION TITLE
List of freelancers/agencies specializing in Filament customization?

### DISCUSSION THREAD

**Discussion by ChrisHardie** [filamentphp/filament#1829] [Ideas]
Does anyone have, and if not would it be helpful to have, a list or directory of freelance developers or agencies available for paid customization and implementation of Filament?

I know I would find this helpful as someone who anticipates a future need to hire those folks in some of my own projects, and as a way of showcasing one part of the awesome ecosystem of people and projects growing around Filament.

I could see it as something linked from the bottom of the marketing site and/or documentation in the same way the sponsorship nudge is, to help people who might read through a doc and realize they are going to need more help, and who are willing to pay for it. Or maybe just a public Google Doc/Sheet folks can add themselves to, to get the list started.

**Comment by ryangjchandler** [comment]
@ChrisHardie I'd highly recommend asking in the Discord server, we could potentially add a new "Jobs" channel too. Feel free to contact me too, ryan [at] digitalbit.uk - I've got some availability coming up soon and could potentially help out.

**Comment by ChrisHardie** [comment]
Thanks @ryangjchandler! FWIW it looks like the Discord invite link on the front page of the marketing site has expired. I'll try again soon.

My own needs are mid- to long-term, and will definitely drop you a note at that time. Also thought it would be helpful to start an easy to find list for others who go looking for the same.

**Reply by danharrin** [reply]
The Discord link isnt broken BTW, Discord itself is down right now.

**Comment by fullyloadedtech** [comment]
@ChrisHardie hello we are interested in business to business opportunities. We are new to Filament but not new to Laravel. I joined the Filament Discord server so see you there or also at https://fullyloaded.tech/ This is a new endeavor and so mid to long-term needs sounds like a good fit. Also interested in helping the development of Filament and/or packages that can easily tie into this system.
